dvclive/live.py

```diff
@@ -194,6 +196,10 @@ def make_report(self):
             if env2bool(env.DVCLIVE_OPEN):
                 open_file_in_browser(self.html_path)
 
+    def make_checkpoint(self):
+        if env2bool(env.DVC_CHECKPOINT) and dvc_checkpoint:
+            dvc_checkpoint()
```

Comment:
We can't really reach this point if we are not in DVC context so we can use `dvc.api` instead of our implementation.

---
`dvc.api.make_checkpoint` also checks if `env.DVC_CHECKPOINT` exists or not but I prefer to have this explicit check here even if it's redundant (also in case `dvc.api` changes).